src/js/components/Pagination/stories/Steps.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Box, Pagination, Text } from 'grommet';
+
+export const Steps = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Box align="start" pad="small" gap="xlarge">
+    <Box fill="horizontal">
+      <Text>summary and stepSelector</Text>
```

Comment:
Aligns with property name

```suggestion
      <Text>summary and stepOptions</Text>
```